test/uk/gov/hmrc/taxhistory/services/EmploymentHistoryServiceSpec.scala

```diff
@@ -161,6 +161,11 @@ class EmploymentHistoryServiceSpec extends UnitSpec with MockitoSugar with TestU
       await(testEmploymentHistoryService.retrieveRtiData(testNino, TaxYear(2016))) shouldBe Some(testRtiData)
     }
 
+    "stub rti employments succeeds with a None" in
+    new StubConnectors(rti = stubRtiGetEmploymentsSucceeds(None)) {
+      await(testEmploymentHistoryService.retrieveRtiData(testNino, TaxYear(2016))) shouldBe None
```

Comment:
Some of these connector specific service methods, such as `retrieveRtiData`, could be made private - we should have tests covering the different stubbed responses on the main service method `retrieveAndBuildPaye` itself.
Perhaps one for a tech debt ticket?

Reply:
Created a ticket for this: https://jira.tools.tax.service.gov.uk/browse/ASA-797